ui/src/app/component/search/search.component.ts

```diff
@@ -78,4 +79,8 @@ export class SearchComponent implements AfterViewInit {
   teamDescriptions(teams: Team[]): string {
     return (teams || []).map(t => t.description).join('\n');
   }
+
+  exportSearchResultToCSV(): void {
+    this.service.exportToCSV(this.searchParams).subscribe(file => saveAs(file,"Search-Results.csv"));
```

Comment:
Could we append a timestamp to the filename? E.g. `DeliusUsers-SearchResults-20210528T120000.csv`.